lib/nixt/respond.js

```diff
@@ -31,7 +25,8 @@ exports.run = function(readable, writable, expects, responses) {
     }
 
     if (typeof expect === 'string') {
-      match = S(buffer).endsWith(expect);
+      var regex = new RegExp(expect+'$');
```

Comment:
Why not `String(buffer).endsWith(expect)`?

Reply:
@nkbt honestly, I didn’t know that existed. I assumed a lib was used because that type of function didn’t exist and yet here it is: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith

It’s nice to learn something new.

Reply:
I’ll make the change.

Reply:
Of course that would be ideal, but it would also require that we lift the engine requirement to ES6, and this package is testing on 0.10. It is not worth it IMO unless some active development will follow (and it doesn't seem like it will).

Reply:
@eush77 agreed, the goal was to keep the scope of change pretty narrow. Alright, your solution would be best. I’ll make the change in a couple hours when it’s daytime here.

Reply:
Even Node 4.0 is "end of life" now, not sure there is any sense in supporting 0.10. https://github.com/nodejs/Release/blob/643af71ee8365efbdccaa134690425ea05931279/README.md#release-schedule.

Reply:
@nkbt if it was a bigger deal I might agree with you more, but the use of `lastIndexOf` isn't a significant complication over `endsWith`